chain/westend-local/config-bob.toml

```diff
@@ -0,0 +1,57 @@
+[global]
+basepath = "~/.gossamer/westend-local-bob"
+log = "info"
+metrics-address = ":9877"
+
+[log]
+core = ""
+network = ""
+rpc = ""
+state = ""
+runtime = ""
+babe = ""
+grandpa = ""
+sync = ""
+digest = ""
+
+[init]
+genesis = "./chain/westend-local/westend-local-spec-raw-v0.9.28.json"
```

Comment:
Update to: genesis = `"./chain/westend-local/westend-local-spec-raw-v0.9.29.json"`